tools/travis/script.sh

```diff
@@ -16,7 +16,11 @@ tools/build_versions.py
 section_end "List.installed.dependencies"
 
 section "Test"
-pytest $TEST_ARGS skimage
+# We can't run it in the git directory since there is a folder called `skimage`
+# in there. pytest will crawl that instead of the module.
+pushd ..
+pytest $TEST_ARGS --pyargs skimage
+popd
```

Comment:
@hmaarrfk can you explain the broader part of your changes? Specifically, the bit above seems like a big change that affects not only the SDIST builds, and `--no-build-isolation` vs `setup.py develop` are different. Could you explain the rationale behind these changes?

Reply:
Did we change the syntax to:

```
(cd ..; pytest ...; cd ..)
```
?

Reply:
```
cd ..; pytest [...]; cd -
```
surely? ;)